app/controllers/logs_controller.rb

```diff
@@ -60,8 +60,9 @@ def api_log_params
   end
 
   def org_params
+    owning_organisation_id = current_user.organisation.holds_own_stock? ? current_user.organisation.id : nil
     {
-      "owning_organisation_id" => current_user.organisation.id,
+      "owning_organisation_id" => owning_organisation_id,
```

Comment:
This change was because we shouldn't set a log's owning organisation as an organisation which doesn't hold stock.